frontend/src/components/phones/dashboard/PhoneDashboard.module.scss

```diff
@@ -39,14 +39,18 @@
   .senders-cta {
     display: flex;
     flex-direction: row;
-    padding: $spacing-xs $spacing-sm;
     border-radius: $border-radius-lg;
     align-items: center;
-    gap: $spacing-xs;
-    margin-left: $spacing-sm;
 
-    &:hover {
-      background: $color-light-gray-10;
+    @media screen and #{$mq-md} {
+      padding: $spacing-sm;
+      gap: $spacing-xs;
+      left: $spacing-sm; // Cancel out extra space from padding
+      position: relative;
```

Comment:
Does this not make more sense, rather than messing around with positioning?

```suggestion
      padding: $spacing-sm;
      padding-right: 0;
      gap: $spacing-xs;
```

Or even:

```suggestion
      padding-block: $spacing-sm;
      gap: $spacing-xs;
```

With potentially a `gap` on the parent, if needed.

Reply:
I did that because I wanted consistent padding around the button, with your implementation it looks like this on hover because of the omitted `padding-right` 
<img width="319" alt="image" src="https://user-images.githubusercontent.com/13066134/186458545-a4f82e00-c1d2-4abc-a89b-f663f9aab89d.png">

Reply:
(that is the hover state on Figma :\ I can talk to Eduardo about keeping hover states simplier so we don't need to do wacky CSS on it)